tests/common/devices/multi_asic.py

```diff
@@ -65,6 +65,7 @@ def critical_services_tracking_list(self):
         service_list = []
         active_asics = self.asics
         if self.sonichost.is_supervisor_node() and self.get_facts()['asic_type'] != 'vs':
+            self._DEFAULT_SERVICES.append("lldp")
```

Comment:
based on this condition, lldp will not be added in DEFAULT_SERVICES for VS supervisor, is that right?
That will be an issue for VS testbed.

Reply:
@SuvarnaMeenakshi : updated.